app/services/StripeService.scala

```diff
@@ -35,13 +35,23 @@ class StripeService(
     intCmp: Option[String],
     ophanPageviewId: String,
     ophanBrowserId: Option[String],
-    idUser: Option[IdentityId]
+    idUser: Option[IdentityId],
+    amount: BigDecimal,
+    currency: String
   ): XorT[Future, String, SavedContributionData] = {
 
     // Fire and forget: we don't want to stop the user flow
     idUser.map { id =>
       identityService.updateMarketingPreferences(id, marketing)
     }
+    emailService.thank(ContributorRow(
```

Comment:
It seems a bit strange to do this only for effect, discarding the XorT for Stripe, but to include it in the result for Paypal.